gcp/service-auth/pyproject.toml

```diff
@@ -0,0 +1,25 @@
+[tool.poetry]
+name = "gcp-service-auth"
+version = "0.1.0"
+description = "Inspecting TeX in arXiv submissions"
+authors = ["arxiv.org"]
+license = "BSD-3 License"
+readme = "README.md"
+packages = [
+    { include = "gcp_service_auth" }
+]
+include = ["README.md"]
+
+[tool.poetry.dependencies]
+python = "^3.8"
```

Comment:
Can we make it work with python 3.11?

Reply:
CIT sycn-node still uses 3.8, and wanted to NOT install python 3.10 on it if I could.

Reply:
> Can we make it work with python 3.11?

It would work any version above 3.8.